wallet/src/test/java/ch/admin/bag/covidcertificate/wallet/inappdelivery/LuhnTest.kt

```diff
@@ -0,0 +1,54 @@
+package ch.admin.bag.covidcertificate.wallet.inappdelivery
+
+import ch.admin.bag.covidcertificate.wallet.inappdelivery.Luhn.TRANSFER_CODE_LEN
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import org.junit.runners.Parameterized.Parameters
+
+
+@RunWith(Parameterized::class)
+class LuhnTest(
+	private val code: CharArray,
+	private val expectedCheckChar: Char,
+) {
+
+	companion object {
+
+		@Parameters
+		@JvmStatic
+		fun buildTestCases(): Collection<Array<Any>> {
+			val cases = listOf<Pair<String, Char>>(
```

Comment:
The sample implementation of the Luhn mod N Wikipedia article gives different check characters for these input values (with the same alphabet of course). Can you explain/check why?